component/libfuse/libfuse2_handler.go

```diff
@@ -160,7 +161,33 @@ func (lf *Libfuse) initFuse() error {
 	if runtime.GOOS == "windows" && lf.networkShare && common.IsDriveLetter(lf.mountPath) {
 		// TODO: We can support any type of valid network share path so this path could
 		// be configurable for the config file. But this is a good default.
-		opts = append(opts, "--VolumePrefix=\\server\\share")
+
+		// by default nameStorage will be blank
+		var nameStorage string
+		var kindStorage string
+		// Borrow bucket-name string from attribute cache
+		if config.IsSet("s3storage.bucket-name") {
+
+			err := config.UnmarshalKey("s3storage.bucket-name", &nameStorage)
+			kindStorage = "bucket"
+			if err != nil {
+				nameStorage = "default"
+				kindStorage = "Cloud Storage"
+				log.Err("ParseAndReadDynamicConfig : Failed to unmarshal s3storage.bucket-name")
+			}
+		} else if config.IsSet("azstorage.container") {
+			err := config.UnmarshalKey("azstorage.container", &nameStorage)
+			kindStorage = "container"
+			if err != nil {
+				nameStorage = "default"
+				kindStorage = "Cloud Storage"
+				log.Err("ParseAndReadDynamicConfig : Failed to unmarshal s3storage.bucket-name")
+			}
+
+		}
```

Comment:
You need an else case here in case they are using our feature where we select the first bucket listed. (This is how our plugin would work.) If it doesn't recognize a container or a bucket then the mount will fail.

Reply:
Are you suggesting to fetch the first bucket list in the else case? or just name the network mount as "cloud default" in that case?